src/schema/scenario-create-cells.json

```diff
@@ -0,0 +1,54 @@
+{
+  "title": "Create Cells Scenario Options",
+  "type": "object",
+  "properties": {
+    "editor": {
+      "title": "Widget type",
+      "description": "Editor widget to test cell creation in",
+      "type": "string",
+      "enum": ["Notebook"],
+      "default": "Notebook"
```

Comment:
In future we could also test creating cells in Console. I don't have time to implement this right now.